### ISSUE TITLE
Allow passing a seed in options

### ISSUE THREAD

**Issue by sporto** [ardeois/graphql-codegen-typescript-mock-data#174]
We would like to add a CI step that checks that the generator matches what we have in the source code (when there are dependency updates of this library). 

For this the output needs to be stable when there are no changes in input schema. So it seems we need some way of setting a constant seed.

As far I can tell there is no way to set a stable seed when using faker.
Is there?

Thanks

**Comment by ardeois** [comment]
@sporto generated mocks should always return the same value if you run the generator multiple times.
We set a seed based on the typename + fieldname (see [here](https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/87b3d1315d2a6f24fd7a752fc494dae6069a375f/src/index.ts#L297))

However, in [major version bump](https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/main/CHANGELOG.md#v400-wed-sep-04-2024), we've changed the default generator value from casual to faker, and the seed for faker changed a little, so that might explain why your generated mock data changed.

If you do generate values dynamically, there is a [seedMock](https://github.com/ardeois/graphql-codegen-typescript-mock-data/tree/main?tab=readme-ov-file#dynamicvalues-boolean-defaultvalue-false) value that can be used as well.
